### PR TITLE
Style tweak

### PR DESCRIPTION
**Pull request by jkrvivian**
Change the font size to fit the template requirement.

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-\documentclass[journal,a4paper]{IEEEtran}
+\documentclass[journal,10pt,a4paper]{IEEEtran}
```
Comment (jserv) [inline paper.tex:1]: Is it consistent to the template specific to IEEE ECICE 2019?

Comment (jkrvivian) [reply paper.tex:1]: No, the template of ECICE has different font size to IEEE tran.

Comment (jserv) [reply paper.tex:1]: I just hesitate with the formation. We should follow the requirements of ECICE 2019.

Comment (jkrvivian) [reply paper.tex:1]: Ok, I'll try another version without the IEEE trans template. 
Not sure the margin and spaces between paragraphs stick to the requirements of ECICE. :thinking: 
What are we going to submit, the pdf or latex file?

Comment (jserv) [reply paper.tex:1]: We would submit PDF only.

Comment (jserv) [reply paper.tex:1]: At the moment, we don't really need to tweak aggressively.

**Comment by jserv** [comment]
Rebase.

**Pull request merged**